diff --git a/dist/html/public.d.ts b/dist/html/public.d.ts
index 7a565572a734e7cc0a7451b388436fc58569ee97..1c062832e891329e50994a518add8e87c08e52da 100644
--- a/dist/html/public.d.ts
+++ b/dist/html/public.d.ts
@@ -1,3 +1,4 @@
 export { childAttributes, createKeyedLazy, createLazy, inertHtml, TextDirection, } from './index.js';
+export { __htmlBuilder as htmlBuilder, __setRuntime as setHtmlRuntime, __clearRuntime as clearHtmlRuntime, } from './index.js';
 export type { Attribute, ChildAttribute, Document, Html, HtmlBuilder, KeyboardModifiers, TagName, } from './index.js';
 //# sourceMappingURL=public.d.ts.map
\ No newline at end of file
diff --git a/dist/html/public.js b/dist/html/public.js
index 42f82c6224657fa6c1bf82bdf397f42776571bdf..4abee391438887a397d504dbf7a66435fcafe25a 100644
--- a/dist/html/public.js
+++ b/dist/html/public.js
@@ -1 +1,2 @@
 export { childAttributes, createKeyedLazy, createLazy, inertHtml, TextDirection, } from './index.js';
+export { __htmlBuilder as htmlBuilder, __setRuntime as setHtmlRuntime, __clearRuntime as clearHtmlRuntime, } from './index.js';
diff --git a/dist/runtime/public.d.ts b/dist/runtime/public.d.ts
index 5ede5ea0d4c87a5ecc7a673820a3cb54b7a9209b..0997cfab0a601571bdfaaad4faa809cf87319826 100644
--- a/dist/runtime/public.d.ts
+++ b/dist/runtime/public.d.ts
@@ -1,4 +1,5 @@
 export { SlowPhase, defaultSlowCallback, embed, makeApplication, makeElement, run, } from './runtime.js';
+export { Dispatch } from './runtime.js';
 export type { RoutingConfig, CrashConfig, CrashContext, ElementCrashConfig, RoutingApplicationConfigWithFlags, RoutingApplicationConfig, ApplicationConfigWithFlags, ApplicationConfig, ElementConfigWithFlags, ElementConfig, ApplicationInit, RoutingApplicationInit, ElementInit, EmbedHandle, InboundPortHandle, InboundPortHandles, OutboundPortHandle, OutboundPortHandles, PortHandles, MakeRuntimeReturn, Visibility, SlowConfig, SlowContext, SlowPatchContext, SlowSubscriptionDependenciesContext, SlowThresholdOverrides, SlowUpdateContext, SlowViewContext, DevToolsConfig, DevToolsMode, DevToolsModeConfig, DevToolsPosition, } from './runtime.js';
 export type { ViewTransitionConfig, ViewTransitionContext, ViewTransitionDecision, } from './viewTransition.js';
 //# sourceMappingURL=public.d.ts.map
\ No newline at end of file
diff --git a/dist/runtime/public.js b/dist/runtime/public.js
index 000c43214a1b7c636034201b5ea8ba54f733ee30..ba9775ad5623b53e7cee5b120db0ca5f6d548e8c 100644
--- a/dist/runtime/public.js
+++ b/dist/runtime/public.js
@@ -1 +1,2 @@
 export { SlowPhase, defaultSlowCallback, embed, makeApplication, makeElement, run, } from './runtime.js';
+export { Dispatch } from './runtime.js';
